### PR TITLE
Setup travis integration

### PR DESCRIPTION
**Pull request by csweaver**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - "8"
+cache:
+  pip: true
+addons:
+  apt:
+    packages:
+    - httpie
+install:
+  - set -eo pipefail
+  - pip install flake8
```
Comment (ttung) [inline .travis.yml]: you could also do pip install httpie here instead of doing it as part of apt.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - ./bin/build-client
+  - python setup.py install
+script:
+  - set -eo pipefail
+  - flake8 server/app/
+  - pytest -s server/test/test_filter.py server/test/test_scanpy_engine.py
+  - cellxgene &
+  - for i in {1..90}; do if http :5005/api/v0.1/initialize > /dev/null; then break; else echo "Waiting for server..."; sleep 1; fi; done
+  - pytest server/test/test_api.py
```
Comment (ttung) [inline .travis.yml]: trailing NL.

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```
Comment (ttung) [inline setup.cfg]: NL.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  - set -eo pipefail
+  - pip install flake8
+  - ./bin/build-client
+  - python setup.py install
+script:
+  - set -eo pipefail
+  - flake8 server/app/
+  - pytest -s server/test/test_filter.py server/test/test_scanpy_engine.py
+  - cellxgene &
+  - for i in {1..90}; do if http :5005/api/v0.1/initialize > /dev/null; then break; else echo "Waiting for server..."; sleep 1; fi; done
```
Comment (ttung) [inline .travis.yml]: You may wish to make this time-based, as that yields a bit more certainty about how long you blow through this loop. :)

File: .travis.yml (hunk 2 above)
Comment (csweaver) [reply .travis.yml]: 🤦‍♀️ Ugh, what is wrong with me. I'll learn eventually...

File: .travis.yml (hunk 1 above)
Comment (csweaver) [reply .travis.yml]: that's cleaner, I like that.

File: .travis.yml (hunk 4 above)
Comment (csweaver) [reply .travis.yml]: Still a loop? Or like sleep 10?

Comment (ttung) [reply .travis.yml]: Oops this should be fine.  I misread this.

**Pull request merged**